client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -143,21 +143,105 @@
     "description": "Feature Flag related to Codegen",
     "features": {
       "useAppsyncModelgenPlugin": {
-        "description": "Allows to use migrated codegen package.",
+        "description": "Allows to use the migrated DataStore model generator package at https://github.com/aws-amplify/amplify-codegen",
```

Comment:
Here and throughout:
1. Rather than "new" and "old", specify the engine, style, or some kind of meaningful distinction that would give a customer some context over why they might choose one style over another.
1. What is "migrated" in reference to: the data, some internal implementation package, or something else?